### DISCUSSION TITLE
Rapid/Autofire

### DISCUSSION THREAD

**Discussion by petecee81** [OnionUI/Onion#1360] [Q&A]
Hi all,

I'm hoping someone can explain (like I’m 5) how to activate autofire when I'm in a game. I've been through the 'turbofire' options in the menu and assigned to every button on the device but with no joy. 

I have also searched for this discussion and found a few things on Reddit with no answers.

Any help would be appreciated. Keep up the great work!

**Comment by Aemiii91** [comment]
What game? What system?

**Reply by petecee81** [reply]
The first game I'm looking to use it in is Carrier Airwing (Arcade)

**Reply by Phanturian** [reply]
Out of curiosity I tried Carrier Airwing with both the FBA 2012 CPS-1 and MAME 2003 Plus cores, and the turbo feature simply doesn't work. I then tried Super Mario Bros (NES) and it doesn't work either. I changed the turbo style between Classic, Toggle, and Hold, and changed the activation to different buttons with no results. I'm under the impression that turbo is listed in the menus but not actually implemented at all.

**Reply by petecee81** [reply]
Yeah I thought I was missing something. Thanks for trying sir. At least now I know it’s not possible.
